package.json

```diff
@@ -19,8 +19,9 @@
     "jsrender": "0.9.84"
   },
   "devDependencies": {
+    "jsreport-core": "^1.5.1",
```

Comment:
add dependence `jsreport-core` here just because we need use the type definition in `jsreport-core`